android/src/main/java/com/swmansion/rnscreens/ScreenStackFragment.kt

```diff
@@ -249,6 +249,18 @@ class ScreenStackFragment :
                 object : WindowInsetsAnimationCompat.Callback(
                     DISPATCH_MODE_STOP,
                 ) {
+                    override fun onPrepare(animation: WindowInsetsAnimationCompat) {
+                        // FormSheet's onResume is called after finishing animator animation but adding
+                        // insets handler then is often too late if we use input autofocus.
+                        // This callback's onPrepare is called before onApplyWindowInsets so we use it
+                        // to set the handler earlier.
+                        (fragment as ScreenStackFragment?)?.sheetDelegate?.let {
```

Comment:
```suggestion
                        this@ScreenStackFragment.sheetDelegate?.let {
```

right? Or do we use some different fragment here?

Reply:
Right, but I moved the code to `Screen` now so I can't really use it there.